.github/workflows/labeling-ticket-done.yml

```diff
@@ -0,0 +1,17 @@
+name: labeling-ticket-done
```

Comment:
Would you please rename these (and the other two you added recently) to use "Sentence case" instead of "kebab-case"?

The jobs list in the GitHub Actions check window is already feeling a bit inconsistent.